01-node-tutorial/answers/globals.js

```diff
@@ -0,0 +1,11 @@
+require("dotenv").config();
```

Comment:
This line should also be removed

Reply:
removed the line as suggested (forgot about it...)